src/Plugin/GraphQL/Schema/ComposableSchema.php

```diff
@@ -76,7 +76,7 @@ public function buildConfigurationForm(array $form, FormStateInterface $form_sta
     $form['extensions'] = [
       '#type' => 'checkboxes',
       '#required' => FALSE,
-      '#title' => t('Enabled extensions'),
+      '#title' => $this->t('Enabled extensions'),
```

Comment:
not sure if this class has the StringTranslationTrait so that this method is available.

Can you test this in local Drupal install by going to the graphql user interface?

Reply:
Ok figured out that it doesn't have that method

Reply:
Ok, then we need to add `use StringTranslationTrait;` to this class.

See for example src/PermissionProvider.php which implements the trait and also has the namespace use statement for it.